go/tools/gopackagesdriver/gopackagesdriver_test.go

```diff
@@ -259,7 +260,117 @@ func TestExternalTests(t *testing.T) {
 	}
 }
 
-func runForTest(t *testing.T, relativeWorkingDir string, args ...string) driverResponse {
+
+func TestOverlay(t *testing.T) {
+	// format filepaths for overlay request using working directory
+	wd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// format filepaths for overlay request
+	helloPath := path.Join(wd, "hello.go")
+	subhelloPath := path.Join(wd, "subhello/subhello.go")
+
+	expectedImportsPerFile := map[string][]string{
+		helloPath: []string{"fmt"},
+		subhelloPath: []string{"os", "encoding/json"},
+	}
+
+	overlayDriverRequest := map[string]map[string]string{
```

Comment:
if you make this type `map[string]map[string][]byte` then you don't have to manually base64 encode when you json serialize it. You're doing a lot of extra logic for something that's built right into the encoder..

Reply:
Updated to use DriverRequest type